net.sourceforge.squirrel_sql.json

```diff
@@ -0,0 +1,88 @@
+{
+  "app-id" : "net.sourceforge.squirrel_sql",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "3.38",
+  "sdk" : "org.gnome.Sdk",
+  "sdk-extensions" : ["org.freedesktop.Sdk.Extension.openjdk11"],
+  "command": "squirrel-sql",
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=x11",
+    "--share=network"
+  ],
+  "modules" : [
+    {
+      "name" : "openjdk",
+      "buildsystem" : "simple",
+      "build-commands" : [ "/usr/lib/sdk/openjdk11/install.sh" ]
+    },
+    {
+      "name" : "jdbc-drivers",
+      "buildsystem" : "simple",
+      "build-commands" : [
+        "mkdir -p /app/jdbc-drivers",
+        "cp * /app/jdbc-drivers/"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://repo1.maven.org/maven2/org/mariadb/jdbc/mariadb-java-client/2.7.2/mariadb-java-client-2.7.2.jar",
+          "sha256": "a3f6776df0842cf65e7f158515012d5307da949f66042282e1ee4474dd19f448"
+        },
+        {
+          "type": "file",
+          "url": "https://repo1.maven.org/maven2/org/xerial/sqlite-jdbc/3.34.0/sqlite-jdbc-3.34.0.jar",
+          "sha256": "605979c94e7fe00437f1e10dcfa657a23f125c8eb4d2f0ec17e3f84613894cc3"
+        },
+        {
+          "type": "file",
+          "url": "https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.19/postgresql-42.2.19.jar",
+          "sha256": "232747fa0924d88a26dfa42b8128b6fa1140802d804125891596e8ca5f291322"
+        },
+        {
+          "type": "file",
+          "url": "https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.23/mysql-connector-java-8.0.23.jar",
+          "sha256": "ff7d5b402afd39c12787471505a33a304103b238ec1b7a44e8936d3329da7535"
+        }
+      ]
+    },
+    {
+      "name": "squirrelsql",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/squirrelsql",
+        "cp -r * /app/squirrelsql",
+        "install -Dm644 icons/acorn.png /app/share/icons/hicolor/512x512/apps/net.sourceforge.squirrel_sql.png"
+      ],
+      "sources": [{
+        "type": "archive",
+        "url": "https://downloads.sourceforge.net/project/squirrel-sql/1-stable/4.1.0-plainzip/squirrelsql-4.1.0-standard.zip",
+        "dest-filename": "squirrelsql-standard.zip",
+        "sha256": "6622f300bc159a51881f74e97efe6fa8388dc48e713a3657b1313305d466275a"
+      }]
+    },
+    {
+      "name": "squirrelsql-startup",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm755 squirrel-sql /app/bin/squirrel-sql",
+        "install -Dm644 net.sourceforge.squirrel_sql.appdata.xml /app/share/metainfo/net.sourceforge.squirrel_sql.appdata.xml",
+        "install -Dm644 net.sourceforge.squirrel_sql.desktop /app/share/applications/net.sourceforge.squirrel_sql.desktop"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "squirrel-sql"
+        },
+        {
+          "type": "file",
+          "path": "net.sourceforge.squirrel_sql.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
I renamed the file to .metainfo.xml